RL-based Maze Solver PR

tushdon2 opened the pull request.

apurba-pp committed:
Add files via upload

apurba-pp committed:
Delete src directory

apurba-pp committed:
Delete src directory

apurba-pp committed:
Merge branch 'tushdon2:main' into main

apurba-pp committed:
Add files via upload

apurba-pp committed:
Delete src directory

apurba-pp committed:
Version 1

Random Maze Solving without implementing RL

yash-bhinwal committed:
Update and rename Code_1.0.ipynb to Code_2.0.ipynb

apurba-pp committed:
Merge pull request #1 from yash-bhinwal/main

Update and rename Code_1.0.ipynb to Code_2.0.ipynb

apurba-pp committed:
Version 3

apurba-pp committed:
Version 4

apurba-pp committed:
Merge pull request #1 from apurba-pp/main

src addition

apurba-pp committed:
Add files via upload

tushdon2 committed:
Adding Images

tushdon2 committed:
Merge pull request #2 from apurba-pp/main

Added Imgprocess.py

apurba-pp committed:
Update ImgPreprocess.py

tushdon2 committed:
Merge pull request #3 from apurba-pp/main

Update ImgPreprocess.py

apurba-pp committed:
Updated

apurba-pp committed:
Updated

apurba-pp committed:
Updated

apurba-pp committed:
Version 4

tushdon2 committed:
Merge pull request #4 from apurba-pp/main

Bola tha Metadata se nhi khelte

tushdon2 committed:
aata majhi satakli

yash-bhinwal committed:
yash ka dimaag sanka

apurba-pp committed:
Merge pull request #2 from yash-bhinwal/main

yash ka dimaag sanka

chirag-ar committed:
Updated

apurba-pp committed:
Merge pull request #3 from chirag-ar/main

Updated

RishabhDubey03 committed:
thak gya vro update karte

apurba-pp committed:
Merge pull request #4 from RishabhDubey03/main

Updated

apurba-pp committed:
Add files via upload

tushdon2 committed:
Merge pull request #5 from apurba-pp/main

Updated

tushdon2 committed:
final src

tushdon2 committed:
definitely final src

tushdon2 committed:
d

RishabhDubey03 committed:
uploaded poster

apurba-pp committed:
Merge pull request #5 from RishabhDubey03/main

uploaded poster

chirag-ar committed:
uploaded report

apurba-pp committed:
Merge pull request #6 from chirag-ar/main

uploaded report

yash-bhinwal committed:
Readme Version 1.0

apurba-pp committed:
Merge pull request #7 from yash-bhinwal/main

Readme Version 1.0

apurba-pp committed:
Add files via upload

tushdon2 committed:
Merge pull request #6 from apurba-pp/main

Updated Readme V2.0, Poster and Report

tushdon2 committed:
README v3.0

tushdon2 committed:
README v4.0

tushdon2 committed:
Uploaded videos

1. Shows the working of Main Jupyter Notebook file
2. Shows the manual process of setting the parameters for image preprocessing

tushdon2 committed:
deleted the video directory

tushdon2 committed:
Added videos

1. Shows the working of Main Jupyter Notebook file
2. Shows the manual process of setting the parameters for image preprocessing

RishabhDubey03 committed:
JPEG FILE UPLOADED

apurba-pp committed:
Merge pull request #8 from RishabhDubey03/main

JPEG FILE UPLOADED

chirag-ar committed:
Final report uploaded

apurba-pp committed:
Delete poster.png

apurba-pp committed:
Merge pull request #9 from chirag-ar/main

Final report uploaded

tushdon2 committed:
Merge pull request #7 from apurba-pp/main

Updated report and poster

tushdon2 committed:
Rename src/Code_4.0.ipynb to Maze_Solver_Code.ipynb

apurba-pp committed:
Delete Image Preprocess.mp4

tushdon2 committed:
Merge pull request #8 from apurba-pp/main

Delete Image Preprocess.mp4

apurba-pp committed:
Update Code_4.0.ipynb

tushdon2 committed:
Add files via upload

tushdon2 committed:
Delete Maze_Solver_Code.ipynb

tushdon2 committed:
Merge pull request #9 from apurba-pp/main

Update Code_4.0.ipynb

tushdon2 committed:
Added Accomplishments

tushdon2 committed:
added ImpPreprocess working

tushdon2 committed:
README v5.0

theobscuredev commented on README.md:
```diff
@@ -1 +1,97 @@
+# RL-based Maze Solver
+Open Projects 2021
+
+![Input Maze Solution](https://github.com/tushdon2/RL-based-Maze-Solver/blob/main/Images%20and%20Videos/Images/Input%20maze1%20and%20solution.png)
+<br>fig1 <i>Input Maze Solution</i>
```
Center align all the images using the `<p>` tag and align attribute. For eg:
`<p align="center">`
  `<img src="https://github.com........." alt="LED Matrix">`
  `<i>LED Matrix</i>`
`</p>`

tushdon2 replied:
centred all the images

theobscuredev commented on README.md:
```diff
@@ -1 +1,97 @@
+
+<!-- ## Electronics Aspect of the Design -->
+
+## Software Aspect of the Design
```
Justify the text for remaining content like Software Aspects, Cost Structure, Limitations, Future Improvements etc which is currently missing.

tushdon2 replied:
justified the paragraphs

tushdon2 replied:
updated the README

theobscuredev reviewed (changes requested):
@tushdon2  Hey, nice work. Please do the suggested changes.

tushdon2 committed:
centered images and justified paragraphs in README

tushdon2 committed:
Update README.md

theobscuredev commented:
@tushdon2 Lemme know once you've made the final changes. Make sure to tag me (@) in this thread.

tushdon2 commented:
@theobscuredev
We have made all the required changes.

theobscuredev commented:
Nice work!
@SanjeevKrishnan I suggest for merge.

SanjeevKrishnan reviewed (changes requested):
Few Changes are required
1. In README.md, under Software Aspect of the Design, add a sub section "Working" and explain the need and working of algorithms used in detail.
2. File names of Images and Videos need to be more generalised. For example in Images and Videos/Videos/Main Program.mp4 doesn't make any sense. I suggest Main Demonstration.mp4 or simply Demonstration.mp4
@theobscuredev Kindly follow this up

tushdon2 committed:
Delete Main Program.mp4

tushdon2 committed:
Added Main Demonstration video

chirag-ar committed:
Final report uploaded

tushdon2 committed:
Added Bellman Equation

apurba-pp committed:
Merge pull request #10 from chirag-ar/main

Final report uploaded

tushdon2 committed:
Merge pull request #10 from apurba-pp/main

Final Report Updated

RishabhDubey03 committed:
workflow1 uploaded

tushdon2 committed:
Animated Solution

tushdon2 committed:
README v6.0

tushdon2 committed:
image processing readme images

chirag-ar committed:
Add files via upload

apurba-pp committed:
Merge pull request #12 from chirag-ar/main

image processing readme uploaded

apurba-pp committed:
Merge branch 'tushdon2:main' into main

tushdon2 committed:
Merge pull request #11 from apurba-pp/main

Added Image Processing Readme file

RishabhDubey03 committed:
Merge branch 'apurba-pp:main' into main

RishabhDubey03 committed:
Add files via upload

apurba-pp committed:
Merge pull request #11 from RishabhDubey03/main

workflow uploaded

tushdon2 committed:
Merge pull request #12 from apurba-pp/main

Workflow image uploaded

yash-bhinwal committed:
Readme v7.0

apurba-pp committed:
Merge pull request #13 from yash-bhinwal/main

Readme v7.0

tushdon2 committed:
Merge pull request #13 from apurba-pp/main

Readme v7.0

tushdon2 commented:
@SanjeevKrishnan @theobscuredev 
The required changes have been made.
Hope we met your expectations. Kindly look into the repo and do the needful.

SanjeevKrishnan reviewed (changes requested):
Few changes are still required!
1. Rename "README ImgPreprocess.md" as "README.md"
2. Rename "Code_4.0.ipynb" as "Maze Solving Code.ipynb"
3. In README.md, do changes as suggested below

SanjeevKrishnan commented on src/README ImgPreprocess.md:
````diff
@@ -0,0 +1,240 @@
+            if count == 2 : return l
+```
+
+#### 9. This [ImgPreprocess.py](https://github.com/tushdon2/RL-based-Maze-Solver/blob/main/src/ImgPreprocess.py) file is then imported by the [Maze Solving Code(Code_4.0.ipynb)](https://github.com/tushdon2/RL-based-Maze-Solver/blob/main/src/Code_4.0.ipynb).
````
Do necessary changes here, after renaming the files as suggested in the previous comment.

tushdon2 replied:
@SanjeevKrishnan  
We have made all the changes you asked for.
Please review.

apurba-pp committed:
Uploaded Code v1.0

defining agent + maze + random maze solving for a given maze without q learning

yash-bhinwal committed:
Updated Code v2.0

maze solving-for a given fixed maze using Q learning

apurba-pp committed:
Merge pull request #14 from yash-bhinwal/main

Updated Code v2.0

apurba-pp committed:
Uploaded Code v3.0

Random maze generator+ solving using Q learning ,without visualisation in matplotlib

yash-bhinwal committed:
Uploaded Code v4.0

Random maze generator+ solving using Q learning ,with visualisation in matplotlib

apurba-pp committed:
Merge pull request #15 from yash-bhinwal/main

Uploaded Code v4.0

RishabhDubey03 committed:
file name changed

RishabhDubey03 committed:
Merge branch 'apurba-pp:main' into main

RishabhDubey03 committed:
Uploaded Code v5.0

Random Maze Generator + Solving using q-learning + arbitrary starting and ending point, with visualization in matplotlib

apurba-pp committed:
Merge pull request #16 from RishabhDubey03/main

Uploaded Code v5.0

apurba-pp committed:
Update README.md

updated link

apurba-pp committed:
Update README.md

Updated Links

chirag-ar committed:
Uploaded Code v6.0

Random Maze Generator + Solving using q-learning + arbitrary starting and ending point, with visualization in matplotlib

apurba-pp committed:
Merge pull request #17 from chirag-ar/main

Uploaded Code v6.0

tushdon2 committed:
Merge pull request #14 from apurba-pp/main

Uploaded Maze Solving Code

tushdon2 committed:
Uploaded Code v7.0

Random Maze Generator+ Input Maze file option +imported ImgPreprocess.py+ Solving using q-learning + arbitrary starting and ending point, with visualization in matplotlib

tushdon2 committed:
Update README.md

tushdon2 committed:
Delete Code_4.0.ipynb

tushdon2 committed:
Update README.md

tushdon2 commented:
@theobscuredev 
@SanjeevKrishnan

SanjeevKrishnan reviewed (approved):
Great Work team!!
PR looks good. Merging the PR
